core/capabilities/webapi/target/target.go

```diff
@@ -0,0 +1,152 @@
+package target
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/types/core"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+	"github.com/smartcontractkit/chainlink/v2/core/capabilities/validation"
+	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/handlers/webapicapabilities"
+)
+
+const ID = "web-api-target@1.0.0"
+
+var _ capabilities.TargetCapability = &Capability{}
+
+var capabilityInfo = capabilities.MustNewCapabilityInfo(
+	ID,
+	capabilities.CapabilityTypeTarget,
+	"A target that sends HTTP requests to external clients via the Chainlink Gateway.",
+)
+
+// Capability is a target capability that sends HTTP requests to external clients via the Chainlink Gateway.
+type Capability struct {
+	capabilityInfo   capabilities.CapabilityInfo
+	connectorHandler *ConnectorHandler
+	lggr             logger.Logger
+	registry         core.CapabilitiesRegistry
+	config           Config
+}
+
+func NewCapability(config Config, registry core.CapabilitiesRegistry, connectorHandler *ConnectorHandler, lggr logger.Logger) (*Capability, error) {
+	return &Capability{
```

Comment:
nit: only fix if doing other fixes, but how about validating some of the fields?  Says connectorHandler is nil?